src/hex/grid.rs

```diff
@@ -54,7 +54,7 @@ impl<TileType> HexGrid<TileType> {
     /// ```
     /// let my_object_pos = (100.0, 432.0);
     /// /// ...
-    /// use gridava::hex::hex_grid::*;
+    /// use gridava::hex::grid::*;
```

Comment:
This is a much better name IMO since hex::hex_grid explicitly duplicates information.